### ISSUE TITLE
Can't join games after it "loads"

### ISSUE THREAD

**Issue by Jeffonehundred** [ValveSoftware/Source-1-Games#5674]
After the game is finished loading the map, aka "sending client info" my game crashes. I can't join ANY game, not even using the create server option works. The game also crashes when I open the Options menu in the main menu.

**Comment by Zoynoob11** [comment]
I've been wanting to log into a server for a while now and every time the game loads it closes for no reason

**Comment by metapea** [comment]
I've been getting he same thing as well

**Comment by metapea** [comment]
No. tf.exe is bootable for starting a map, but then say "You are in insecure mode. You must restart before you can connect to secure servers." when joining any server like with #5680 
There's also no launch options for the 32bit version far as i can tell

**Comment by jd498269** [comment]
I have this same exact problem. I somehow managed to get an item drop anyway after crashing and relaunching lol.

No joining servers or opening options from the main menu

Also unable to load training levels

**Comment by LoveLama** [comment]
Try to join on community servers, and after join to the official casual, it works for me  👍🏾

**Comment by doonv** [comment]
If you get an error relating to insecure mode, try adding `-steam` to your launch options.

**Comment by jd498269** [comment]
I think most people in here tried community servers as well. At least I did and it's the same thing. There are no errors. It just crashes poof.

**Comment by Jeffonehundred** [comment]
@jd498269 Yes. Tht is if you use the default Tf2 exe "tf_win64.exe". If you use "tf.exe" you get the "You are in an insecure mode. You must restart before you can connect to secure servers." error. (There is someone else  with this error on github, I do not know how to link another issue in a thread, could someone tell me the syntax for it?). It does not give a single reason on why it crashes. Is there a log somewhere I can check in the game files? Like in Minecraft?

**Comment by Jeffonehundred** [comment]
I think this crashing issues with "tf_win64.exe" might be with the new unicode text rendering. The game might be trying to launch with setting spots with emojis in mind, but windows 7 and 8.1 do not have emojis built into the os. Just a theory, but ill download a bunch of emojis and different fonts to my pc and see if its help. I have a windows 7 backup made, just in case something brakes.

**Comment by Jeffonehundred** [comment]
> I think this crashing issues with "tf_win64.exe" might be with the new unicode text rendering. The game might be trying to launch with setting spots with emojis in mind, but windows 7 and 8.1 do not have emojis built into the os. Just a theory, but ill download a bunch of emojis and different fonts to my pc and see if its help. I have a windows 7 backup made, just in case something brakes.

**Comment by Jeffonehundred** [comment]
@markosreal so far no. I can only get Unicode 6.0 to work on my pc, and its only in some programs. I have not found that it had worked. Still trying with different methods. Any ideas about what I should try and do?

**Comment by Jeffonehundred** [comment]
is:open label:"Team Fortress 2"

**Comment by Jeffonehundred** [comment]
how do I make labels on posts? I can't seem to find away to label this as Widows and Team fortress 2.

**Comment by kisak-valve** [comment]
tf2 crashed when join a server 

Issue transferred from https://github.com/ValveSoftware/Source-1-Games/issues/5826.
@2xCelebrations posted on 2024-04-20T14:49:56:

[crash_tf_win64.exe_20240420220050_1.dmp](https://github.com/ValveSoftware/Source-1-Games/files/15048635/crash_tf_win64.exe_20240420220050_1.dmp)
***
@NULLYUKI commented on 2024-04-20T14:51:32:

Which Windows version are you running?
Might be related to https://github.com/ValveSoftware/Source-1-Games/issues/5674

**Comment by kisak-valve** [comment]
For reference, the attached minidump is an EXCEPTION_ACCESS_VIOLATION_READ in winmm.dll on Windows 6.1.7601 (7 Service pack 1).

**Comment by instafixer** [comment]
Hello!
Try adding this launch option and then launch TF2 via steam library:
"D:\SteamLibrary\steamapps\common\Team Fortress 2\tf.exe" %command% -steam

Let me know if it fixed the issue.

**Comment by Jeffonehundred** [comment]
@instafixer I'll try this right away

**Comment by Jeffonehundred** [comment]
@instafixer No, this command does not work at all, Steam gives the error "Windows cannot find 'A:\SteamLibrary\steamapps\common\Team'. Make sure you typed the name correctly, and then try again. A: is where my game install is, but it looks like this command is getting hung up at the space where Team^Fortress is.

**Comment by instafixer** [comment]
@Jeffonehundred Just to confirm, did you include the quotations marks as well? This is necessary.

Your launch options should be exactly like this:
"A:\SteamLibrary\steamapps\common\Team Fortress 2\tf.exe" %command% -steam

If it isn't, double check what your path to tf.exe and/or tf_win64.exe and adjust is accordingly

**Comment by Jeffonehundred** [comment]
Oh, I had removed the quotes! I feel like such a greenbeard. I tried it on Windows 7, and IT WORKS! @instafixer I can't tahank you enough. I would say to close this thread, but unfortunately the 64 bit build (tf_win64.exe) is the one having the crash issue at the moment. As for the 32 bit bulid (tf.exe) does work with this command ["A:\SteamLibrary\steamapps\common\Team Fortress 2\tf.exe" %command% -steam].

**Comment by jd498269** [comment]
Yup. Forcing the 32-bit version with the `%command% -steam` commands "fixes" the problem. Hope the 64-bit version gets some revisions. So it's still an issue. Hope to experience the 64-bit version for myself.

**Comment by kawasakinnn** [comment]
@Jeffonehundred what emoji packs? can you send the link for them?

**Comment by Jeffonehundred** [comment]
@kawasakinnn I used this guide, then this file "Seguiemj.tff"

**Comment by NULLYUKI** [comment]
According to patch [220641](https://www.teamfortress.com/post.php?id=220641)
`Fixed a crash when opening options/going into game (mostly affecting Windows 7 users)`

This issue requires retesting.

**Comment by Jeffonehundred** [comment]
Ok, I have done testing for over 30 mints in one setting. I can confirm there are no more crashes or bugs. I can open the settings menu and join games properly.  If I have any issues ill let you all know. For now, this issue can be closed.

**Comment by jd498269** [comment]
yea im just here to post an update that the new 64 bit exe has been updated and is working fine on my machine. THANKS VALVE DEVS! hope this is a step in the right direction towards thwarting the bot abuse since the source code leak. i understand you guys are working against the tide and appreciate the effort you're putting into a game which many would leave to rot. most dont understand the challenge you guys are facing and expect the bot problem to be fixed overnight. thanks again for giving this game a nice coat of polish.

**Comment by stumpid** [comment]
Yeah, I'm sorry for necroposting, but I have to say that this problem has reoccurred on windows 11! It works for me to open tf2, but if I open settings it either crashes my game or removes all my sound! If I join a casual match or a community server, they both crash to desktop. I'm using the command that was posted in this thread, it's not working, but it does get closer to a solution when I use it.

**Comment by kosetnoi** [comment]
Hi, sorry for writing in a "dead"/fixed bug post. But I've had the exact same problems described here again on windows 11.
1. Crashing upon pressing the options button
2. Server loading screen forever stuck on "sending client info", then crashing.

I was wondering if anyone has had a similar situation on win11 and managed to crack the code in solving these 2 problems?

**Comment by kosetnoi** [comment]
I just want to say that if anyone comes across this issue like I have and stumbles upon this thread, I managed to fix both issues via disabling the windows game bar.

I repeat the windows game bar caused the issue, I looked into the security and maintenance logs on windows and noticed that on the same day that my TF2 started having these 2 issues, the gamebar stopped working as well.

The solution I used was looking up this "Multi-Plane Overlay disable registry fix" in my search engine, and following the steps to disable it, after following the steps and restarting my PC, I managed to fix the issue.
